docs/installation.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: doc
+---
+
+# ImapTest Installation
+
+## Compiling
+
+### Dovecot CE
+
+Download and compile [Dovecot CE sources](https://www.dovecot.org/download/). Use the most recent version possible (nightly snapshots are best).
+
+::: info
+ImapTest uses Dovecot's library functions.
+
+Dovecot doesn't have any external dependencies.
+:::
+
+1. Compiling goes the usual way: `./configure && make`.
+   * There's no need for `make install`.
+   * ImapTest configure wants to find `dovecot-config`, which is created by running (and finishing) `make`.
+   * If you want to avoid having to install shared Dovecot libraries, you can use `./configure --without-shared-libs`.
+
+### ImapTest
+
+1. Download and compile [ImapTest](https://dovecot.org/nightly/imaptest/imaptest-latest.tar.gz).
```

Comment:
Well .. we don't have these anymore updated. I guess should point to github.com/dovecot/imaptest/ and above point to download github.com/dovecot/core/

And since it's no longer in tarball, need to add ./autogen.sh to instructions here.

Reply:
OK, pointed exclusively to github sites for source.